cluster/images/provider-gcp/Dockerfile

```diff
@@ -5,11 +5,12 @@ ARG TARGETOS
 ARG TARGETARCH
 ARG CROSSPLANE_PROVIDER_VERSION
 
-ENV USER_ID=65532
+ENV USER_ID 65532
```

Comment:
🙈 Douh I did the reverse of the WARNING message.

```
 1 warning found (use docker --debug to expand):
 - LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 10)
 ```
 
 I'll fix it, thanks for catching.

Reply:
😆 Thanks for all your efforts